data/campaigns/Eastern_Invasion/scenarios/11_Captured.cfg

```diff
@@ -714,6 +718,7 @@
                 condition=win
             [/objective]
             [objective]
+                #po: Initially shown with 4 escapees.
```

Comment:
any language where the plural form is different when number is 4? Otherwise, useless

Reply:
The same string will also collect the po hint "Later shown meaning all of the released prisoners. Some can die, just not all."

4 being different than 3 or 5 seems to be possible in `cs`, `lt`, `pl`, `sk`, `sl`.

4 being different than 20 seems to be possible in `cs`, `pl` and `sk`.